internal/nodes/avi_model_nodes_utils.go

```diff
@@ -121,6 +126,14 @@ func (o *AviVsNodeCommonFields) ConvertToRef() {
 		if o.ApplicationProfileRef != nil {
 			o.ApplicationProfileRef = proto.String("/api/applicationprofile?name=" + *o.ApplicationProfileRef)
 		}
+		SslKeyAndCertificateRefs := make([]string, 0, len(o.SslKeyAndCertificateRefs))
```

Comment:
These are the objects already present in Avi controller. Do we need to translate those to `/api/sslkeyandcerticiate` format?

Reply:
We are specifying just the sslkeycert name in l4rule of an already existing sskkeycert obj in avi, and I think it should be converted to ref just like we do for every other object.